.circleci/config.yml

```diff
@@ -109,9 +108,9 @@ commands:
     steps:
       - run:
           command: |
-            echo $PWD
-            PERL5OPT="$PERL5OPT -MDevel::Cover=-ignore,^/|^utils/|^x?t/|\.lttc$,-blib,0" starman -I$HOME/project/lib -I$HOME/project/old/lib \
-                    --preload-app --pid starman.pid --port 5762 --workers 2 \
+            PERL5OPT="$PERL5OPT -MDevel::Cover=-ignore,^/|^utils/|^x?t/|\.lttc$,-blib,0" \
```

Comment:
What's wrong with the starman setup that we had? (and more importantly: Why do we need this *on this PR*?)

Reply:
1- The line should have been shown as splitted.
2- I can move to Gazelle instead of Starman in another PR

Reply:
ok, but I meant to ask why we should be switching to Gazelle.

Reply:
Test speedup, based on https://github.com/kazeburo/Gazelle/wiki/Benchmark

Reply:
But that comparison almost exclusively compared request handling overhead. Isn't it overhead elsewhere?

Reply:
Gazelle tests on CircleCI in 31:45 minutes, starman, 34:34. Both were successful on PhantomJS, See https://app.circleci.com/pipelines/github/ylavoie/LedgerSMB/808/workflows/b8c46aa0-a9f8-40da-b820-ab8e9f11156a/jobs/3113 vs https://app.circleci.com/pipelines/github/ylavoie/LedgerSMB/809/workflows/36409fb7-b6b4-4203-ab49-979e214ffbc0/jobs/3112
We are running Starlet on Travis, Gazelle would do fine there too, if we find a 10% reduction worth enough.